### PR TITLE
Feat/initialize backend

### PR DESCRIPTION
**Pull request by vade123**
- local dev env for azure functions with a dummy health check func 
- local mongodb
- somewhat sensible eslint, prettier and ts configs based on previous projects

### CODE REVIEW

**Review by Kitsupitsu — APPROVED** [review]

File: backend/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "dist",
+    "rootDir": ".",
+    "sourceMap": true,
+    "strict": true,
+    "allowSyntheticDefaultImports": true,
+    "declaration": false,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
```
Comment (skumpulainen) [inline backend/tsconfig.json:12]: front end can definitely share some of these. I would probably add `noUnusedLocals` and `noUnusedParameters `

File: backend/prettier.config.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  semi: false,
+  printWidth: 100,
```
Comment (skumpulainen) [inline backend/prettier.config.js:3]: could add `"bracketSpacing": true` and`  "bracketSameLine": false`

File: backend/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- [azure-functions-core-tools@4](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-typescript)
+- Docker for local development environment
+
+# Getting started
+
+```bash
+$ npm install
+$ npm start
+```
```
Comment (skumpulainen) [inline backend/README.md:15]: Here could be some instructions to activate the correct node version via local `.nvmrc,` for example using `nvm`.

**Review by skumpulainen — APPROVED** [review]

**Comment by hollannikas** [comment]
Converted to draft while looking into [this azd template](url)

**Pull request closed without merging**